tests/js/client/aql/vector/aql-vector-agency-wrong-definition-cluster.js

```diff
@@ -0,0 +1,200 @@
+// / WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// / See the License for the specific language governing permissions and
+// / limitations under the License.
+// /
+// / Copyright holder is ArangoDB GmbH, Cologne, Germany
+// /
+/// @author Jure Bajic
+// //////////////////////////////////////////////////////////////////////////////
+
+const internal = require("internal");
+const jsunity = require("jsunity");
+const arangodb = require("@arangodb");
+const helper = require("@arangodb/aql-helper");
+const aql = arangodb.aql;
+const getQueryResults = helper.getQueryResults;
+const assertQueryError = helper.assertQueryError;
+const errors = internal.errors;
+const db = internal.db;
+const {
+    randomNumberGeneratorFloat,
+} = require("@arangodb/testutils/seededRandom");
+const dbName = "vectorDb";
+const collName = "vectorColl";
+const indexName = "vectorIndex";
+let {
+    agency,
+    getMetric,
+    getEndpointById
+} = require('@arangodb/test-helper');
+
+let IM = global.instanceManager;
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief test suite
+////////////////////////////////////////////////////////////////////////////////
+
+function VectorIndexCorrectDefinitionInAgencyTest() {
+    let collection;
+    const dimension = 500;
+    const seed = 12132390894;
+
+    return {
+        setUp: function() {
+            db._createDatabase(dbName);
+            db._useDatabase(dbName);
+
+            collection = db._create(collName, {
+                numberOfShards: 3
+            });
+
+            let docs = [];
+            let gen = randomNumberGeneratorFloat(seed);
+            for (let i = 0; i < 100; ++i) {
+                const vector = Array.from({
+                    length: dimension
+                }, () => gen());
+                docs.push({
+                    vector
+                });
+            }
+            collection.insert(docs);
+        },
+
+        tearDown: function() {
+            db._useDatabase("_system");
+            db._dropDatabase(dbName);
+        },
+
+        testVectorIndexCreationCorrect: function() {
+            let beforeVersion = IM.agencyMgr.get("Current/Version").arango.Current.Version;
+            collection.ensureIndex({
+                "name": "test",
+                "params": {
+                    "dimension": dimension,
+                    "metric": "l2",
+                    "nLists": 1,
+                },
+                "fields": ["vector"],
+                "type": "vector"
+            });
+
+            // wait for servers to pick up the changes
+            internal.sleep(5);
+
+            let afterVersion = IM.agencyMgr.get("Current/Version").arango.Current.Version;
+
+          // If the index is created we do not expect too much version change
+          // Unfortunately this is the only indicator that we have not entered
+          assertTrue(afterVersion - beforeVersion < 10);
```

Comment:
This test sounds like a recipe for unstability in the CI. Let's see if we can come up with something better. Here, we do not expect anything to go wrong, even though `nProbes` is not set, since the index creation process will set a default anyway. So it should be good enough to see if the index creation did not throw and in the end, getIndexes shows the index. I suggest we remove the sleep and the Current version test here.